debezium-server/debezium-server-infinispan/src/test/java/io/debezium/server/infinispan/InfinispanTestResourceLifecycleManager.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.server.infinispan;
+
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.atomic.AtomicBoolean;
+
+import org.infinispan.client.hotrod.impl.ConfigurationProperties;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.testcontainers.containers.BindMode;
+import org.testcontainers.containers.GenericContainer;
+
+import io.quarkus.test.common.QuarkusTestResourceLifecycleManager;
+
+public class InfinispanTestResourceLifecycleManager implements QuarkusTestResourceLifecycleManager {
```

Comment:
One question - is it better to use container ot start the server in-process?
Also could be the tag in the image name synchronized with Infinispan version?

Reply:
> One question - is it better to use container ot start the server in-process? 

IMHO better to use the container - it's more close to real use case and also I'm actually not sure if connecting to Infinispan started in library mode via hotrod is supported (or even possible).

> Also could be the tag in the image name synchronized with Infinispan version?

ops, fixed